cpp/include/cudf/io/parquet.hpp

```diff
@@ -510,6 +527,24 @@ class parquet_writer_options {
   {
     _column_chunks_file_path.assign(file_path);
   }
+
+  /**
+   * @brief Sets the maximum row group size, in bytes.
+   */
+  void set_row_group_size_bytes(size_t size_bytes)
+  {
+    CUDF_EXPECTS(size_bytes >= 512 << 10, "512KB is the minimum rowgorup size");
+    _row_group_size_bytes = size_bytes;
+  }
+
+  /**
+   * @brief Sets the maximum row group size, in rows.
```

Comment:
Are 512 KB / 5000 rows arbitrary or are they standards defined somewhere? I would like to see a reference linked in a comment.

Reply:
There are no standards for this (AFAIK). I chose these values because row groups consist of pages, and this is the expected size of each page. We could also make the page size configurable, but it's a lot more involved so I left it as-is until there's a customer ask.
I can expand the error messages to mention that the minimum size is the page size.